src/pg/test/expected/40_observatory_utility_test.out

```diff
@@ -0,0 +1,79 @@
+SELECT set_config(
```

Comment:
@ohasselblad I see you added some fixtures and expectations to the test dir but no test file.